clang/lib/DPCT/PatternRewriter.cpp

```diff
@@ -282,6 +282,16 @@ static int parseCodeElement(const MatchPattern &Suffix,
   int Index = Start;
   const int Size = Input.size();
   while (Index >= 0 && Index < Size) {
+
+    if (dpct::DpctGlobalInfo::getBuildScript() == BuildScript::BS_Cmake ||
```

Comment:
I think the if condition does not tell input file is cmake file?  it's not good to check extra condiion in the leaf processing function, suggest to have another flag to tell the content of input is cmake file.

Reply:
Okay, will updated in the next updated PR.